app/AppKernel.php

```diff
@@ -32,26 +28,54 @@ public function registerBundles()
             new JMS\DiExtraBundle\JMSDiExtraBundle(),
             new JMS\AopBundle\JMSAopBundle(),
             new Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle(),
+            new Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle(),
+            new Nelmio\ApiDocBundle\NelmioApiDocBundle(),
 
             new Os2Display\MediaBundle\Os2DisplayMediaBundle(),
             new Os2Display\CoreBundle\Os2DisplayCoreBundle(),
             new Os2Display\AdminBundle\Os2DisplayAdminBundle(),
             new Os2Display\DefaultTemplateBundle\Os2DisplayDefaultTemplateBundle(),
-            new Os2Display\VimeoBundle\Os2DisplayVimeoBundle(),
```

Comment:
Vimeo og Youtube skal ikke længere være med? 
Eller ligger de et andet sted?

Reply:
Jeg fjernede dem fra reload repoet for at holde det så tæt på upstream som muligt, tænker at tilføje dem til bbs repoet.